Dockerfile

```diff
@@ -72,6 +72,8 @@ ARG GIT_TREE_STATE
 RUN mkdir -p ui/dist
 COPY --from=argo-ui ui/dist/app ui/dist/app
 
+# Ensure in github CI that we don't attempt to rebuild this file
+RUN touch ui/dist/app/index.html
```

Comment:
```suggestion
# update timestamp so that `make` doesn't try to rebuild this -- it was already built in the previous stage
RUN touch ui/dist/app/index.html

```
- move this up next to the `argo-ui` `COPY` statement
- be more specific in the comment